packages/engine/Specs/Scene/Model/PntsLoaderSpec.js

```diff
@@ -538,6 +540,18 @@ describe(
       });
     });
 
+    it("loads PointCloudDracoInvalid without crashing", async function () {
```

Comment:
This is a total nitpick, but can we make the test name a bit more descriptive of the context?

Maybe something along the lines of the following.

```suggestion
    it("loads legacy point cloud metadata with Draco compression", async function () {
```